netcore/Makefile

```diff
@@ -129,48 +129,15 @@ xtest-%: prepare update-corefx-tests
 		-notrait category=outerloop -notrait category=nonosxtests -notrait category=failing \
 		-notrait category=nonnetcoreapptests -notrait category=nonmonotests \
 		-html ../../TestResult-$*.html -nunit ../../TestResult-$*-netcore-xunit.xml \
-		$(shell if [ -a $(CURDIR)/excludes-$*.rsp ]; then grep -v '^#\|^$$' $(CURDIR)/excludes-$*.rsp; fi;) \
+		$(shell if [ -a $(CURDIR)/CoreFX.issues.rsp ]; then grep -v '^#\|^$$' $(CURDIR)/CoreFX.issues.rsp; fi;) \
```

Comment:
Can we specify it as response file now (`@CoreFX.issues.rsp`)? I don't know if the fix for non-lower-case file names was flowed back to Mono yet, but it may have been. Alternatively we can just name it `corefx.issues.rsp` for now.

Reply:
@filipnavara the idea is to move to the `RunScript.sh`. I am not sure if I can lowercase that file, don't remember where it's set on CI. @ViktorHofer can we have a custom name for it?

Reply:
I see. The properly cased file name may work now. We should test it, and perhaps bump CoreFX version if necessary.